### PR TITLE
Coming up

### LINKED ISSUE #67: Coming up section desktop - LIVE SITE
- [ ] Develop this section, ComingUpSection or UpcomingSection for DESKTOP only (mobile will be separate)
- [ ] Implement the designs for when there are no events, 1, 2, and 3 events displayed. 
Screenshot 2022-11-30 at 12 20 44 AM

### PR DESCRIPTION
**Pull request by jhylisa666**
Fixes #67 .

Changelist:

- added coming-up component

Notes:

- only for desktop

Screenshots & Screencasts:
https://user-images.githubusercontent.com/92692037/205935382-9e3e6555-5e39-4caf-a0ed-2f958353d495.mov

### CODE REVIEW

File: live-site/components/coming-up/ComingUp.styles.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+`;
+
+const StyledEvents = styled.div`
+display: flex;
+justify-content: center;
+
+`;
+
+const StyledHeader = styled.div`
+color: WHITE;
```
Comment (jzhzhang66) [inline live-site/components/coming-up/ComingUp.styles.tsx]: use colors.WHITE

File: live-site/components/coming-up/ComingUp.styles.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+export {
+    StyledEvent,
+    StyledHeader, 
+    StyledTime, 
+    StyledBody, 
+    StyledSectionContainer, 
+    StyledEvents, 
+    StyledTextContainer, 
+    StyledSectionHeader
+}
```
Comment (jzhzhang66) [inline live-site/components/coming-up/ComingUp.styles.tsx]: run formatter on all your files

File: live-site/components/coming-up/ComingUp.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  StyledHeader,
+  StyledTime,
+  StyledBody, 
+  StyledSectionContainer, 
+  StyledEvents, 
+  StyledTextContainer, 
+  StyledSectionHeader
+} from './ComingUp.styles';
+
+// need coming up props because want to check that if it's empty then....
```
Comment (jzhzhang66) [inline live-site/components/coming-up/ComingUp.tsx]: remove this comment it's irrelevant now

File: live-site/components/coming-up/ComingUp.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    header: 'Register your team',
+    time: 'Complete by 12:00am EST',
+    body: 'hello hello hello hello hello hello hello hello hi hi hello hello hello hi hi'
+  };
+  const event3: UpcomingEvent = {
+    header: 'Register your team',
+    time: 'Complete by 12:00am EST',
+    body: 'hello hello hello hello hello hello hello hello hi hi hello hello hello hi hi'
+  };
+  const events: UpcomingEvent[] = [event1, event2, event3];
```
Comment (jzhzhang66) [inline live-site/components/coming-up/ComingUp.tsx]: if these 3 events are all the same can't u just do [event, event, event]?

File: live-site/components/coming-up/ComingUp.tsx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+  if (events.length == 0) {
```
Comment (jzhzhang66) [inline live-site/components/coming-up/ComingUp.tsx]: use === equal signs for deep equality

File: live-site/components/coming-up/ComingUp.tsx [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    return <NoUpcoming />;
+  } else {
+    return (
+      <StyledSectionContainer>
+        <StyledSectionHeader>Coming up...</StyledSectionHeader>
+        <StyledEvents>
+        {events.map((event: UpcomingEvent) => (
+          <StyledEvent>
```
Comment (jzhzhang66) [inline live-site/components/coming-up/ComingUp.tsx]: make sure ur events have an id so you can add key={event.id}

File: live-site/components/coming-up/no-upcoming/NoUpcoming.tsx [hunk 7]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { NoUpcomingContainer, StyledH2 } from './NoUpcoming.styles';
+
+// need coming up props because want to check that if it's empty then....
+const NoUpcoming: React.FC = () => {
+  // state --> which list of events
+  // eventsList is list of list of events grouped in the number
```
Comment (jzhzhang66) [inline live-site/components/coming-up/no-upcoming/NoUpcoming.tsx]: remove all comments in this file

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
This looks great Lisa!!!! Amazing work thanks for working on this bae<333 

I left a few minor comments and also it looks like the font isn't being applied if you could look into that + fix it!<333
Screenshot 2022-12-06 at 6 15 37 PM

Screenshot 2022-12-06 at 6 15 32 PM

**Comment by jzhzhang66** [comment]
Also please resolve merge conflicts before re-requesting !

File: live-site/components/coming-up/ComingUp.styles.tsx [hunk 8]
```diff
@@ -0,0 +1,73 @@
+import styled from "styled-components";
+import { max } from "../../../shared-ui/lib/responsive";
+import { colors } from "../../../shared-ui/style/colors";
+import { fonts, H3 } from "../../../shared-ui/style/typography";
+
+const StyledSectionContainer = styled.div`
+padding-top: 10em;
+
```
Comment (Dhivas-Sugumar) [inline live-site/components/coming-up/ComingUp.styles.tsx]: remove extra lines

**Review by Dhivas-Sugumar — CHANGES_REQUESTED** [review]
LGTM except one small comment and judy's comments

**Review by jzhzhang66 — APPROVED** [review]
APPROVEDDD

**Review by Dhivas-Sugumar — APPROVED** [review]
Approved !

**Pull request merged**